src/components/30-organisms/footer/index.scss

```diff
@@ -205,7 +205,7 @@
       background-color: $color-azure;
 
       &-list-item {
-        @include typo(primary, xx-small, default);
+        @include text(t4);
```

Comment:
"xx-small" is different to t4 size. So... do it look good even with t4? Did you talk to a designer because of this design change?

Reply:
We have talked a lot about this. t4 is the smallest available so there is no other possibility. I have talked to a designer yesterday.

Reply:
I don't know what xx-small was, but there is no visible change